crates/bevy_text/src/text.rs

```diff
@@ -165,22 +165,22 @@ impl TextLayout {
 # use bevy_color::Color;
 # use bevy_color::palettes::basic::{RED, BLUE};
 # use bevy_ecs::World;
-# use bevy_text::{Font, TextLayout, TextStyle, TextSection};
+# use bevy_text::{Font, TextLayout, TextFont, TextSection};
 
 # let font_handle: Handle<Font> = Default::default();
 # let mut world = World::default();
 #
 world.spawn((
     TextLayout::default(),
-    TextStyle {
+    TextFont {
         font: font_handle.clone().into(),
         font_size: 60.0,
         color: BLUE.into(),
```

Comment:
Feel like CI should have caught this one

Reply:
doc tests need to use with `///` not `/* */` blocks